psd-web/db/migrate/20200225103836_enable_pgcrypto.rb

```diff
@@ -0,0 +1,7 @@
+class EnablePgcrypto < ActiveRecord::Migration[5.2]
+  def change
+    safety_assured do
+      enable_extension "pgcrypto"
```

Comment:
Might be an idea to guard against the extension already being installed?

```suggestion
      enable_extension "pgcrypto" unless extension_enabled?("pgcrypto")
```

Reply:
There is no need: https://github.com/rails/rails/blob/f33d52c95217212cbacc8d5e44b5a8e3cdc6f5b3/activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb#L386

Reply:
doh!